### PR TITLE
Team Management

### PR DESCRIPTION
**Pull request by yingrong1996**
did Team management

### CODE REVIEW

File: src/seedu/addressbook/commands/HelpCommand.java [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package seedu.addressbook.commands;
 
 
+import seedu.addressbook.commands.Team.*;
```
Comment (E0176106) [inline src/seedu/addressbook/commands/HelpCommand.java]: import explicitly

File: src/seedu/addressbook/commands/Team/AddTeam.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
+package seedu.addressbook.commands.Team;
+
+import seedu.addressbook.commands.Command;
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.team.*;
```
Comment (E0176106) [inline src/seedu/addressbook/commands/Team/AddTeam.java]: import explicitly

File: src/seedu/addressbook/commands/Team/AddTeam.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        return toAdd;
+    }
+
+    @Override
+    public CommandResult execute() {
+        try {
+            addressBook.addTeam(toAdd);
+            return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+        } catch (UniqueTeamList.DuplicateTeamException dpe) {
+            return new CommandResult(MESSAGE_DUPLICATE_PERSON);
```
Comment (E0176106) [inline src/seedu/addressbook/commands/Team/AddTeam.java]: person changed to team

File: src/seedu/addressbook/commands/Team/EditTeam.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+//package seedu.addressbook.commands.Team;
```
Comment (E0176106) [inline src/seedu/addressbook/commands/Team/EditTeam.java]: Junk

File: src/seedu/addressbook/commands/Team/FindTeam.java [hunk 5]
```diff
@@ -0,0 +1,55 @@
+package seedu.addressbook.commands.Team;
+
+import seedu.addressbook.commands.Command;
+import seedu.addressbook.commands.CommandResult;
+import seedu.addressbook.data.team.ReadOnlyTeam;
+
+import java.util.*;
```
Comment (E0176106) [inline src/seedu/addressbook/commands/Team/FindTeam.java]: import explicitly

File: src/seedu/addressbook/data/AddressBook.java [hunk 6]
```diff
@@ -58,20 +67,40 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
...
+
+
     /**
      * Defensively copied UniquePersonList of all persons in the address book at the time of the call.
      */
     public UniquePersonList getAllPersons() {
         return new UniquePersonList(allPersons);
     }
 
+    public UniqueTeamList getAllTeams() { return new UniqueTeamList(allTeams); }
```
Comment (E0176106) [inline src/seedu/addressbook/data/AddressBook.java]: coding standard violation

File: src/seedu/addressbook/data/team/PlayerList.java [hunk 7]
```diff
@@ -0,0 +1,13 @@
+//package seedu.addressbook.data.team;
```
Comment (E0176106) [inline src/seedu/addressbook/data/team/PlayerList.java]: Junk

File: src/seedu/addressbook/data/team/UniqueTeamList.java [hunk 8]
```diff
@@ -0,0 +1,134 @@
+package seedu.addressbook.data.team;
+
+
+import seedu.addressbook.common.Utils;
+import seedu.addressbook.data.exception.DuplicateDataException;
+
+import java.util.*;
```
Comment (E0176106) [inline src/seedu/addressbook/data/team/UniqueTeamList.java]: import explicitly

**Comment by huyidi** [comment]
good effort.

**Pull request merged**